### ISSUE TITLE
PreDB Error

### ISSUE THREAD

**Issue by lostandlost** [Radarr/Radarr#1322]
When I click the PreDB Sync button or run the PreDB system task I get the following error in my logs: 
Non 200 StatusCode Forbidden encountered while searching PreDB.

I also noticed that items that are on the PreDB are not marked in Radarr as available.  The two may or may not be related. 

Radarr Ver. 0.2.0.535

**Comment by MostDefiantly** [comment]
I too have the same issue, same version as well. 

Just a guess, but requests originating from Radarr clients may have been throttled on PreDB's side. Same thing happened with MusicBrainz and Headphones. 

It's been almost a month since the last develop branch release, perhaps this has been addressed in dailies somewhere along the line.

**Comment by Salamasond** [comment]
I just came to search as I have the same issue via manual or auto sync:

Radarr Ver. 0.2.0.535
Mono JIT compiler version 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)

`17-4-6 08:29:04.4|Info|PreDBService|Starting PreDB Sync
17-4-6 08:29:04.5|Trace|EventAggregator|Publishing CommandUpdatedEvent
17-4-6 08:29:04.5|Trace|EventAggregator|CommandUpdatedEvent -> CommandModule
17-4-6 08:29:04.5|Trace|EventAggregator|CommandUpdatedEvent <- CommandModule
17-4-6 08:29:04.5|Trace|HttpClient|Req: [GET] http://predb.me?rss=1&cats=movies
17-4-6 08:29:04.5|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
17-4-6 08:29:05.1|Trace|Http|Res: 548 [GET] /api/diskspace: 200.OK (74003 ms)
17-4-6 08:29:05.1|Trace|Http|Res: 549 [GET] /api/diskspace: 200.OK (73981 ms)
17-4-6 08:29:05.1|Debug|Api|[GET] /api/diskspace: 200.OK (74003 ms)
17-4-6 08:29:05.1|Debug|Api|[GET] /api/diskspace: 200.OK (73981 ms)
17-4-6 08:29:05.2|Trace|HttpClient|Res: [GET] http://predb.me?rss=1&cats=movies: 403.Forbidden (743 ms)
17-4-6 08:29:05.2|Warn|PreDBService|Non 200 StatusCode Forbidden encountered while searching PreDB.
17-4-6 08:29:05.2|Trace|EventAggregator|Publishing PreDBSyncCompleteEvent`

**Comment by jasonbrent** [comment]
Ditto - preDB errors as above.

**Comment by galli-leo** [comment]
@MostDefiantly Do you know if and how they circumvented that?

**Comment by MostDefiantly** [comment]
@galli-leo They never really did, judging by the errors I still receive refreshing or adding artist metadata. It was really bad when Headphones was the hot shit. Pretty sure the Dev group could never work out a deal with MusicBrainz so the requests had to be[ throttled back cause it slaughtered MB's traffic](https://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting). Headphones was pretty much unusable during this time, but it's cleared up a lot since then - probably cause less people are using the software, tbh. 

Only workaround I know of was running your own local/cloud hosted sub-instance of MusicBrainz, pay someone (or Rembo for VIP creds) to do the same, or just let it take forever to gather all the data by the Nth retry. :)

**Comment by gold007eye** [comment]
Latest nightly here with the same issue.

**Comment by evilvibes** [comment]
Also having the issue. 
![error](https://cloud.githubusercontent.com/assets/17504733/24778946/9974685a-1aea-11e7-897c-8a5c98c3bbf9.png)

**Comment by PeteOdeath** [comment]
Same problem here. All my movies are set for Minimum Avaliability: PreDB
So should I change them to something else?

**Comment by lyrad87** [comment]
@PeteOdeath Try "Announced" for the time being. Seems to fix the issue for me even though I'd still prefer PreDB to work correctly.

**Comment by InAnimaTe** [comment]
Yeah it would be really cool if we could get a community run mirror cluster going so we only have one (or few) source(s) hitting predb etc... 

There's no way we're going to be able to continue using PreDB long-term like this. It'd be nice to also have multiple predb's we tap into as well.

**Comment by jamesonp** [comment]
Same issue

**Comment by rjsears** [comment]
Same issue here as everyone else...

**Comment by mtr2014** [comment]
Me too...

**Comment by arm3n** [comment]
Ditto.

**Comment by Firemad03** [comment]
#1369 I got more information in the error. I think my bug is related to this one :)

**Comment by galli-leo** [comment]
Currently not much we can do, since predb apparently just blocks Radarr clients now. When I have got more time I want to talk to the maintainers of predb.

**Comment by some-guy-23** [comment]
I noticed that PreDB is using CloudFlare in front of their site. It could be they are inadvertently blocking the request, viewing it as a DDOS attack - https://support.cloudflare.com/hc/en-us/articles/200169226-Why-am-I-getting-a-403-error-

Other thing I'd note is, right now PreDB is not using caching for their RSS page that is being requested - http://predb.me/?rss=1&cats=movies

If they enabled CloudFlare caching for this URL (using PageRules), this would go a long ways towards reducing the server load coming from Radarr instances. Could make it essentially negligible.

**Comment by alfablac** [comment]
What's the best setup for this?
Just realized that I missed John Wick Chapter 2 because I had set up to PreDB min avail.
As soon as I changed it to In Cinema, Radarr grabbed correctly according to my custom filter.
Is that the best setup to avoid fakes? Had to narrow the size slide because I saw a fake release of Guardians of the Galaxy vol.2 on my indexers feed (but Radarr didn't grab it, not sure why)
